src/home/HomePage.test.tsx

```diff
@@ -30,17 +30,18 @@ describe('HomePage', () => {
     expect(screen.getByTestId('home-banner')).toBeInTheDocument();
   });
 
-  it('should pass showHomepagePromoVideo and homepagePromoVideoYoutubeId to HomeBanner and open the video modal', () => {
+  it('opens video modal with YouTube iframe when video button is clicked', async () => {
```

Comment:
[inform]: The name of this test has been updated to reflect what this test tests.